jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/transport/HttpConversation.java

```diff
@@ -32,115 +35,162 @@ public class HttpConversation extends Attributes.Lazy
-     * This list changes as the conversation proceeds, as follows:
+     * <p>Updates the response listeners, eventually using the given override response listener,
+     * that must be notified instead of the response listeners of the first exchange.</p>
+     * <p>The response listeners change as the conversation proceeds, as follows:</p>
      * <ol>
      * <li>
-     * request R1 send =&gt; conversation.updateResponseListeners(null)
+     * request R1 send {@code => conversation.updateResponseListeners(null)}
      * <ul>
      * <li>exchanges in conversation: E1</li>
      * <li>listeners to be notified: E1.listeners</li>
      * </ul>
      * </li>
      * <li>
-     * response R1 arrived, 401 =&gt; conversation.updateResponseListeners(AuthenticationProtocolHandler.listener)
+     * response R1 arrived, 401 {@code => conversation.updateResponseListeners(AuthenticationProtocolHandler.listener)}
      * <ul>
      * <li>exchanges in conversation: E1</li>
      * <li>listeners to be notified: AuthenticationProtocolHandler.listener</li>
      * </ul>
      * </li>
      * <li>
-     * request R2 send =&gt; conversation.updateResponseListeners(null)
+     * request R2 send {@code => conversation.updateResponseListeners(null)}
      * <ul>
      * <li>exchanges in conversation: E1 + E2</li>
      * <li>listeners to be notified: E2.listeners + E1.listeners</li>
      * </ul>
      * </li>
      * <li>
-     * response R2 arrived, 302 =&gt; conversation.updateResponseListeners(RedirectProtocolHandler.listener)
+     * response R2 arrived, 302 {@code => conversation.updateResponseListeners(RedirectProtocolHandler.listener)}
      * <ul>
      * <li>exchanges in conversation: E1 + E2</li>
      * <li>listeners to be notified: E2.listeners + RedirectProtocolHandler.listener</li>
      * </ul>
      * </li>
      * <li>
-     * request R3 send =&gt; conversation.updateResponseListeners(null)
+     * request R3 send {@code => conversation.updateResponseListeners(null)}
      * <ul>
      * <li>exchanges in conversation: E1 + E2 + E3</li>
      * <li>listeners to be notified: E3.listeners + E1.listeners</li>
      * </ul>
      * </li>
      * <li>
-     * response R3 arrived, 200 =&gt; conversation.updateResponseListeners(null)
+     * response R3 arrived, 200 {@code => conversation.updateResponseListeners(null)}
      * <ul>
      * <li>exchanges in conversation: E1 + E2 + E3</li>
      * <li>listeners to be notified: E3.listeners + E1.listeners</li>
      * </ul>
      * </li>
      * </ol>
-     * Basically the override conversation listener replaces the first exchange response listener,
-     * and we also notify the last exchange response listeners (if it's not also the first).
-     *
-     * This scheme allows for protocol handlers to not worry about other protocol handlers, or to worry
+     * <p>Basically the override conversation listener replaces the first exchange response listener,
```

Comment:
This still doesn't explain it very well.  Also the chatty style of "Basically" is not good.